src/Microsoft.Extensions.Http.Polly/HttpRequestMessageExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Http;
+using System;
+using System.Net.Http;
+
+namespace Polly
```

Comment:
My goal is to avoid polluting System.Net.Http. We could also use `Microsoft.Extensions.Http` but it seems more natural to use the Polly namespace since you'd likely have it already added if you're interested in the context. Thoughts?

Reply:
With the metapackage approach, it means that a lot of users have will have this package available without realizing it. This is ultimately going to be included in the `Microsoft.AspNetCore.All` package.

Reply:
Re:
>goal is to avoid polluting System.Net.Http. [...] seems more natural to use the Polly namespace

:+1: So the extension methods would not appear unless someone had `using Polly;`, not confusing those not interested.